benches/README.md

```diff
@@ -0,0 +1,28 @@
+# Bevy Benchmarks
+
+This is a crate with a collection of benchmarks for Bevy, separate from the rest of the Bevy crates.
+
+## Running the benchmarks
+
+1. If cargo isn't already installed, [install it](https://www.rust-lang.org/tools/install).
+2. Move into the `benches` directory (where this README is located).
+
+    ```sh
+    bevy $ cd benches
```

Comment:
No to mention `cd` also exists on [Windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd), which is the only other major OS I could think of this affecting.

Reply:
I don't feel strongly enough about this to hold up the PR, but my argument is that `bevy $` might look sort of arcane to a Windows user. It might not be obvious that this is not part of the command they need to type. And we already have text describing which directory to be in.

Reply:
I wouldn't block on that either. I agree that it can be misunderstood by someone, also I haven't see this "style" to be used anywhere before in the project.